.github/workflows/ci.yml

```diff
@@ -1,4 +1,6 @@
-on: push
+on:
+  - push
+  - pull_request
```

Comment:
Note, for pull requests this triggers duplicate builds (one for the push, another one for the pr event).

Also, it builds for any push to any branch.

Not sure if you want this, usually it is enough to only run CI for PRs targeting the main branch, and run on pushes to the main branch:

```
on:
  push:
    branches: master
  pull_request:
    branches: master
```

Just my 2 ¢

Reply:
Thanks @avdv :pray: